ci/scripts/run-build-ic.sh

```diff
@@ -39,6 +38,10 @@ if [[ "${RUN_ON_DIFF_ONLY:-}" == "true" ]]; then
     fi
 
     if [ ${#ARGS[@]} -eq 1 ]; then
+        if [ "${IS_PROTECTED_BRANCH:-}" == "true" ]; then
```

Comment:
@marko-k0 what do you think of this? No functional changes, but would protect against this sort of issue in the future.

Reply:
Maybe.